### DISCUSSION TITLE
Default group for new users?

### DISCUSSION THREAD

**Discussion by tamasgal** [snipe/snipe-it#13193] [Q&A] [answered]
Is there a way to assign a default group for each newly created user? We use an LDAP server and want each user to be able to check out assets on their own and we want them to be in the `Editor` group automatically.

**Comment by devvonmonahan** [comment, marked as answer]
It's currently a feature on master in the ldap settings - "Default Permissions Group"

![image](https://github.com/snipe/snipe-it/assets/102929115/811df711-abd6-474c-82a8-327148bb8c74)

It was added as part of v6.1.0, read pull request #11736

**Reply by tamasgal** [reply]
Perfect, thanks!
